src/system/api/app/include/gunwipc.h

```diff
@@ -82,21 +98,67 @@ extern enum gnwIpcError ipcRegister(const char * const path,
                                     const bool bindingRequired,
                                     const size_t permissionMask);
 
+/*
+    Registers the process as IPC notification receiver for given path
+
+    Params:
+        * path - IPC path (see line 14)
```

Comment:
```suggestion
        * path - IPC path (see line 14)
```
without prefix?

Podobnie jak dla w komenatrzu dla `ipcSend`. Być może warto by było wprowadzić określenie dla "absolutnej ściezki" która zawiera prefiks oraz sciężkę, oraz jakiś inny termin dla określenia samej ścieżki, gdy takowa jest wymagana

Reply:
Być może warto by było wprowadzić alias dla typu który zawiera ścieżkę dla IPC

Reply:
Myślę, że będzie to temat do dalszej dyskusji, ponieważ w architekturze IPC każda ścieżka jest w sumie bezwzględna (choć prefix może na to nie wskazywać)

Niemniej poprawiłem komentarz aby był bardziej jasny

Reply:
> Być może warto by było wprowadzić alias dla typu który zawiera ścieżkę dla IPC

In progress

Reply:
> Być może warto by było wprowadzić alias dla typu który zawiera ścieżkę dla IPC

Gotowe